tests/test_packageAlerts.py

```diff
@@ -0,0 +1,389 @@
+        self.exposure.setDetector(detector)
+
+        visit = afwImage.VisitInfo(
+            exposureId=1234,
+            exposureTime=200.,
+            date=dafBase.DateTime("2014-05-13T17:00:00.000000000",
+                                  dafBase.DateTime.Timescale.TAI))
+        self.exposure.getInfo().setVisitInfo(visit)
+
+        self.filter_names = ["g"]
+        afwImageUtils.resetFilters()
+        afwImageUtils.defineFilter('g', lambdaEff=487, alias="g.MP9401")
+        self.exposure.setFilter(afwImage.Filter('g'))
+
+        diaObjects = makeDiaObjects(2, self.exposure)
+        diaSourceHistory = makeDiaSources(10,
+                                          diaObjects["diaObjectId"],
+                                          self.exposure)
+        self.diaObjects, diaSourceHistory = _roundTripThroughApdb(
+            diaObjects,
+            diaSourceHistory,
+            self.exposure.getInfo().getVisitInfo().getDate().toPython())
+        self.diaObjects.replace(to_replace=[None], value=np.nan, inplace=True)
+        diaSourceHistory.replace(to_replace=[None], value=np.nan, inplace=True)
+        diaSourceHistory["programId"] = 0
+        diaSourceHistory["ra_decl_Cov"] = None
+        diaSourceHistory["x_y_Cov"] = None
+        diaSourceHistory["ps_Cov"] = None
+        diaSourceHistory["trail_Cov"] = None
+        diaSourceHistory["dip_Cov"] = None
+        diaSourceHistory["i_cov"] = None
+
+        self.diaObjects["ra_decl_Cov"] = None
+        self.diaObjects["pm_parallax_Cov"] = None
+        self.diaObjects["uLcPeriodic"] = None
+        self.diaObjects["gLcPeriodic"] = None
+        self.diaObjects["rLcPeriodic"] = None
+        self.diaObjects["iLcPeriodic"] = None
+        self.diaObjects["zLcPeriodic"] = None
+        self.diaObjects["yLcPeriodic"] = None
+        self.diaObjects["uLcNonPeriodic"] = None
+        self.diaObjects["gLcNonPeriodic"] = None
+        self.diaObjects["rLcNonPeriodic"] = None
+        self.diaObjects["iLcNonPeriodic"] = None
+        self.diaObjects["zLcNonPeriodic"] = None
+        self.diaObjects["yLcNonPeriodic"] = None
+
+        self.diaSources = diaSourceHistory.loc[
+            [(0, "g", 8), (1, "g", 9)], :]
+        self.diaSourceHistory = diaSourceHistory.drop(labels=[(0, "g", 8),
+                                                              (1, "g", 9)])
+
+    def testMakeCutoutBytes(self):
+        """Test round tripping an exposure/cutout to bytes and back.
+        """
+        packageAlerts = PackageAlertsTask()
+
+        sphPoint = self.exposure.getWcs().pixelToSky(self.center)
+        cutout = self.exposure.getCutout(sphPoint, packageAlerts.cutoutBBox)
+
+        cutoutBytes = packageAlerts.makeCutoutBytes(cutout)
+        tempMemFile = afwFits.MemFileManager(len(cutoutBytes))
+        tempMemFile.setData(cutoutBytes, len(cutoutBytes))
+        cutoutFromBytes = afwImage.ExposureF(tempMemFile)
+        self.assertTrue(cutout, cutoutFromBytes)
```

Comment:
I don't think I understand why this is `assertTrue`, and not `assertEqual` or something. What is this testing?

Reply:
Bug on my part. Will change to assertTrue.